soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -16,17 +16,21 @@ def create
 
   def destroy
     playlist = Playlist.find(params[:id])
-
-    delete_group_redirect(playlist)
+    delete_group_invites(playlist)
+    redirect_to dashboard_path
   end
 
   private
 
-  def delete_group_redirect(playlist)
-    playlist.group_playlists.destroy_all
-    playlist.playlist_songs.destroy_all
-    playlist.destroy
-    redirect_to dashboard_path
+  def delete_group_invites(playlist)
+    group = playlist.groups[0]
```

Comment:
Actually this is the rabbit hole I went down yesterday by changing the schema. Yeah turns out I need that there for deleting bc playlist belong to a user so I need two playlist for a group(assuming the group is two strong).  Rabbit whole(https://github.com/Jlawlzz/personal-project/pull/77)